.github/workflows/lint.yml

```diff
@@ -49,10 +49,15 @@ jobs:
         with:
           toolchain: stable
           components: rustfmt
+      - name: Update PATH
+        run: |
+          echo "$HOME/.local/bin" >> "$GITHUB_PATH"
```

Comment:
Actually, I would prefer to add the new dir to the front of PATH env, but I don't know how to do it.

Reply:
I think `export GITHUB_PATH="$HOME/.local/bin:$GITHUB_PATH"`

Reply:
I guess here $GITHUB_PATH is a file.

Reply:
I see it's a file